src/gui/dialogs/window_dlg_quickpause.cpp

```diff
@@ -24,5 +24,12 @@ DialogQuickPause::DialogQuickPause(fsm::BaseData)
         marlin_vars()->media_LFN.copy_to(buff, FILE_NAME_BUFFER_LEN, lock);
         gcode_name.SetText(_(buff));
     }
+
+    const char *msg;
+    memcpy(&msg, (uint32_t *)data.GetData().data(), sizeof(uint32_t));
+    if (msg) {
+        text.SetText(_(msg));
```

Comment:
We do not want to translate this string. Use `string_view_utf8::MakeRAM()` for construction.

Reply:
Done.